README.md

```diff
@@ -37,7 +37,7 @@ In its current release these are the benchmarks that are executed:
   - `mongodb`: GNU AGPL v3.0 (http://www.mongodb.org/about/licensing/)
   - `ycsb` (used by `mongodb`): Apache V2 (https://github.com/brianfrankcooper/YCSB/blob/master/LICENSE.txt)
   - `memtier_benchmark`: GPL v2 (https://github.com/RedisLabs/memtier_benchmark)
-  - `sysbench_oltp`: GPL v2 (https://launchpad.net/sysbench)
+  - `sysbench_oltp`: GPL v2 (https://github.com/akopytov/sysbench)
```

Comment:
Whoops, I could have made life a little bit easier with a reference there.  And sorry about the spam pull.  :-)